### PR TITLE
Feature/add view controller menu #5

### PR DESCRIPTION
**Pull request by suho**

### CODE REVIEW

File: FourSquare/Controllers/Base/PageBase/MenuItemViewController.swift [hunk 1]
```diff
@@ -64,7 +62,6 @@ extension MenuItemViewController: UITableViewDataSource {
 
 extension MenuItemViewController: UITableViewDelegate {
     func tableView(tableView: UITableView, heightForRowAtIndexPath indexPath: NSIndexPath) -> CGFloat {
-        let rowHeight: CGFloat = 140
         return rowHeight
```
Comment (le-binh) [inline FourSquare/Controllers/Base/PageBase/MenuItemViewController.swift]: if all rows have the same row height, prefer use `rowHeight` property of `UITableView`

File: FourSquare/Controllers/Home/HomeViewController.swift [hunk 2]
```diff
@@ -37,12 +37,22 @@ class HomeViewController: BaseViewController {
     var pageMenu: CAPSPageMenu?
     var itemViewControllers: [UIViewController] = []
     var activeMenuItems: [ItemMenu] = []
+    var topPicksViewController: TopPicksViewController?
```
Comment (le-binh) [inline FourSquare/Controllers/Home/HomeViewController.swift]: Why optional?

File: FourSquare/Controllers/AppDelegate/AppDelegate.swift [hunk 3]
```diff
@@ -9,22 +9,33 @@
...
 class AppDelegate: UIResponder, UIApplicationDelegate {
 
     var window: UIWindow?
 
+    var homeViewController: HomeViewController?
+
     func application(application: UIApplication, didFinishLaunchingWithOptions launchOptions: [NSObject: AnyObject]?) -> Bool {
         setup()
         window = UIWindow(frame: UIScreen.mainScreen().bounds)
+        self.setUpGoogleMapAPIKey()
```
Comment (le-binh) [inline FourSquare/Controllers/AppDelegate/AppDelegate.swift]: move to `setupThirdParties`

File: FourSquare/Controllers/Base/PageBase/MenuItemViewController.swift (hunk 1 above)
Comment (suho) [inline FourSquare/Controllers/Base/PageBase/MenuItemViewController.swift]: fixed

File: FourSquare/Controllers/Home/HomeViewController.swift (hunk 2 above)
Comment (suho) [inline FourSquare/Controllers/Home/HomeViewController.swift]: fixed

File: FourSquare/Controllers/AppDelegate/AppDelegate.swift (hunk 3 above)
Comment (suho) [inline FourSquare/Controllers/AppDelegate/AppDelegate.swift]: fixed

**Pull request merged**